assignment-2/xwalker/assignment-2.rb

```diff
@@ -0,0 +1,120 @@
+# 1. Customize the room_cleaner program.
+# Possibilities:
+#
+# Accept user input to get room names (or offer a selection of available rooms)
+# Create an invoice-style output at the end of the program that varies based on the rooms and processes completed.
+
+$cleaning_array = []
+
+def get_task
+  puts "Name of the room you'd like for me to clean? (yes/no)"
```

Comment:
I didn't mean to put (yes/no) here. It's just asking for input on the room the user would like to clean. :/